mkdocs.yml

```diff
@@ -170,4 +170,5 @@ nav:
       - 'Templates': 'templates.md'
       - 'Frequently Asked Questions': 'faq.md'
       - 'Known Limitations': 'known-limitations.md'
+  - Security and Permissions: 'security-and-permissions.md'
```

Comment:
The new security content I created and that @rarkins reviewed and merged into the main repository is not yet live: https://docs.renovatebot.com/security-and-permissions/

So maybe you want to wait until the Git submodule updates with the new content before putting the page in the sidebar?

Reply:
Maybe nest it under Other?